### PR TITLE
Update p1 model to 2020A

### PR DESCRIPTION
**Pull request by cquiroz**
This PR updates the p1 model in ITAC for 2020A and includes a few minor fix suggested by IntelliJ

### CODE REVIEW

File: phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java [hunk 1]
```diff
@@ -145,11 +145,11 @@
     protected SubmissionReceipt receipt = null;
     protected SubmissionAccept accept = null;
     protected SubmissionRequest request = null;
-    protected SubmissionRequest band3Request = null;
+    private SubmissionRequest band3Request = null;
     protected Itac itac = null;
-    protected String programId = null;
-    protected Map<Partner, SubmissionRequest> requests = new HashMap<Partner, SubmissionRequest>();
-    protected Set<Keyword> keywords = new HashSet<Keyword>();
+    private String programId = null;
```
Comment (swalker2m) [inline phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java]: Why are some `private` and others `protected`?  Seems like they should all be `private`?

File: phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java [hunk 2]
```diff
@@ -82,7 +82,7 @@
             dummyPartnerBR,
             dummyPartnerCA,
             dummyPartnerCL,
-            dummyPartnerGS,
+            dummyPartnerKR,
```
Comment (swalker2m) [inline phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java]: What happened to GS?

File: phase1-data/src/test/scala/edu/gemini/tac/persistence/phase1/ProposalMigratorTest.scala [hunk 3]
```diff
@@ -200,59 +199,58 @@ class ProposalMigratorTest {
...
-  def toBlueprint(site: String,  r: Node) = {
+  def toBlueprint(site: String,  r: Node): BlueprintBase = {
     site match {
-      case "Gemini North"   => {
-          val laser = (r\"resourceComp" filter (rc => (rc\"resourceCompName" text) contains("Laser guide star") )) length
-          val hasLaser = if (laser > 0) true else false
-          if (hasLaser) ProposalBuilder.GNIRS_IMAGING_H2_PS05_LGS else ProposalBuilder.GMOS_N_IMAGING
-        }
+      case "Gemini North"   =>
+        val laser = (r\"resourceComp" filter (rc => (rc\"resourceCompName" text) contains "Laser guide star" )) length
```
Comment (swalker2m) [inline phase1-data/src/test/scala/edu/gemini/tac/persistence/phase1/ProposalMigratorTest.scala]: I guess this hasn't really changed from before, but I think omitting the `.` has fallen out of style.

File: phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java (hunk 2 above)
Comment (cquiroz) [reply phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java]: GS was removed from the list of partners several semesters back. GS now submits as US. I saw this on a test and removed it in a pinch

File: pom.xml [hunk 4]
```diff
@@ -34,7 +34,7 @@
         <unitils.version>3.1</unitils.version>
         <commons.dbcp.version>1.4</commons.dbcp.version>
         <commons.httpclient.version>3.1</commons.httpclient.version>
-        <ocs.model.p1.version>2019102.1.0</ocs.model.p1.version>
+        <ocs.model.p1.version>2020001.1.0</ocs.model.p1.version>
```
Comment (cquiroz) [inline pom.xml]: This is the key change. We need to update the p1 model

File: phase1-data/src/test/scala/edu/gemini/tac/persistence/phase1/ProposalMigratorTest.scala (hunk 3 above)
Comment (cquiroz) [reply phase1-data/src/test/scala/edu/gemini/tac/persistence/phase1/ProposalMigratorTest.scala]: Yeah, this is old scala. I think even newer versions won't take this xml parsing style?
In essence these are suggestions from IntelliJ to remove parentheses

File: phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java (hunk 1 above)
Comment (cquiroz) [reply phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java]: Again IntelliJ suggestions. It detects that something can be made private.
We could actually remove this whole fixtures thing. We haven't run these unit tests in years

**Pull request merged**